dataprofiler/data_readers/data_utils.py

```diff
@@ -611,7 +657,8 @@ def load_as_str_from_file(
             while start_loc < len_sample_lines - 1 and total_occurrences < max_lines:
                 loc, occurrence = find_nth_loc(
                     sample_lines[start_loc:],
-                    search_query=search_query_value,
+                    search_query=cast(str, search_query_value),
```

Comment:
is this fixed and not needing casting if we fix to accept bytes?

Reply:
See https://github.com/capitalone/DataProfiler/pull/662#discussion_r998298127